### PR TITLE
docs: add basic docs for browsers

### PR DESCRIPTION
**Pull request by OrKoN**

### CODE REVIEW

File: packages/browsers/src/CLI.ts [hunk 1]
```diff
@@ -149,6 +176,22 @@ export class CLI {
             desc: 'Search for a browser installed on the system instead of the cache folder.',
             default: false,
           });
+          yargs.example(
+            '$0 launch chrome@1083080',
+            'Launch the Chrome browser identified by the revision 1083080.'
+          );
+          yargs.example(
+            '$0 launch chrome@112.0a1',
```
Comment (Lightning00Blade) [inline packages/browsers/src/CLI.ts]:
```suggestion
            '$0 launch firefox@112.0a1',
```

File: packages/browsers/src/CLI.ts [hunk 2]
```diff
@@ -77,27 +78,53 @@ export class CLI {
...
     });
   }
 
   async run(argv: string[]): Promise<void> {
     await yargs(hideBin(argv))
+      .scriptName('@puppeteer/browsers')
       .command(
         'install <browser>',
-        'Download and install the specified browser',
+        'Download and install the specified browser. If successful, the command outputs the actual browser buildId that was installed and the absolute path to the browser executable.',
```
Comment (Lightning00Blade) [inline packages/browsers/src/CLI.ts]: What do you think about explicitly telling what the pattern of the line is `<browser>@<buildID>  <path>` ?

File: packages/browsers/src/CLI.ts [hunk 3]
```diff
@@ -61,7 +61,8 @@ export class CLI {
 
   #defineBrowserParameter(yargs: yargs.Argv<unknown>): void {
     yargs.positional('browser', {
-      description: 'The browser version',
+      description:
+        'Which browser to install <browser>[@<buildId|latest>]. `latest` will try to find the latest available build. `buildId` is a browser-specific identifier such as a version or a revision.',
```
Comment (Lightning00Blade) [inline packages/browsers/src/CLI.ts:65]: This command is used for both `install` and `launch` but the second does not have a `latest`. Maybe we should input the description as an argument?

Comment (OrKoN) [reply packages/browsers/src/CLI.ts:65]: This is listed in known limitations. Eventually, we would probably support latest in both.

File: packages/browsers/src/CLI.ts (hunk 2 above)
Comment (OrKoN) [reply packages/browsers/src/CLI.ts]: Done

**Review by Lightning00Blade — APPROVED** [review]

**Pull request merged**